upstream/ocaml_flambda/utils/config.common.ml

```diff
@@ -1,182 +0,0 @@
-#2 "utils/config.common.ml"
```

Comment:
I think something got messed up here

Reply:
`config.common.ml` got replaced with `config.common.ml.in` in flambda. I deleted this file, and added `upstream/ocaml_flambda/configure`, which is the file that now contains the magic numbers